provider/cldr/src/transform/datetime/week_data.rs

```diff
@@ -0,0 +1,339 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use crate::error::Error;
+use crate::reader::open_reader;
+use crate::support::KeyedDataProvider;
+use crate::CldrPaths;
+use icu_calendar::arithmetic::week_of::CalendarInfo;
+use icu_datetime::provider::*;
+use icu_provider::iter::IterableProvider;
+use icu_provider::prelude::*;
+use std::borrow::Cow;
+use std::collections::HashSet;
+use std::convert::TryFrom;
+use tinystr::TinyStr4;
+
+/// All keys that this module is able to produce.
+pub const ALL_KEYS: [ResourceKey; 1] = [key::WEEK_DATA_V1];
+
+mod cldr_json {
```

Comment:
Nit: cldr_json definitions should be moved into the common module at the top level of the crate.

Reply:
Moved into cldr_serde